core/src/main/java/hudson/model/UpdateSite.java

```diff
@@ -243,9 +243,23 @@ private FormValidation verifySignature(JSONObject o) throws IOException {
      * Let sub-classes of UpdateSite provide their own signature validator.
      * @return the signature validator.
      */
+    @Deprecated
```

Comment:
Also add <code>@deprecated</code> Javadoc with a pointer to the replacement method